dll/win32/shell32/wine/control.c

```diff
@@ -736,6 +737,107 @@ static	void	Control_DoWindow(CPanel* panel, HWND hWnd, HINSTANCE hInst)
 }
 #endif
 
+#ifdef __REACTOS__
+
+/** Structure for in and out data when
+ *  search for the cpl dialog of first instance
+ */
+typedef struct tagAppDlgFindData
+{
+    PCWSTR  szAppFile;  /**< Full path to applet library as search parameter */
+    signed  sAppletNo;  /**< Number of applet in a system control library as search parameter */
```

Comment:
normally a number of elements is not signed (and the type would be something like UINT or something similar),